src/ott/tools/soft_sort.py

```diff
@@ -450,6 +451,84 @@ def _quantile(
   return apply_on_axis(_quantile, inputs, axis, q, weight, **kwargs)
 
 
+def multiv_cdf_quantile_maps(
+    inputs: jnp.ndarray,
+    targets: Optional[jnp.ndarray] = None,
+    target_sampler: Optional[Callable[[jax.random.PRNGKey, Tuple[int, int]],
+                                      jnp.ndarray]] = None,
+    key: Optional[jax.random.PRNGKey] = None,
+    target_num_samples: Optional[int] = None,
+    cost_fn: Optional[jnp.ndarray] = None,
+    epsilon: Optional[float] = None,
+    input_weights: Optional[jnp.ndarray] = None,
+    target_weights: Optional[jnp.ndarray] = None,
+    **kwargs: Any
+) -> [
```

Comment:
For clarity, I'd use, e.g., `... -> Tuple[Func_t, Func_t]` and define `Func_t = Callable[[jnp.ndarray], jnp.ndarray]`